shared/containers/AllLayout.js

```diff
@@ -0,0 +1,20 @@
+import MapLayout from './mapLayout';
```

Comment:
It's really good job. But I think this is component, not container.
So I think it should be located on 'shared/component/AllLayout.js'
and on 'app.js' import just component since we don't need container.